Adapters/Vungle/ISVungleAdapter/ISVungleRewardedVideoAdapterRouter.m

```diff
@@ -0,0 +1,127 @@
+//
+//  ISVungleRewardedVideoAdapterRouter.m
+//  ISVungleAdapter
+//
+//  Copyright © 2020 ironSource. All rights reserved.
+//
+
+#import "ISVungleRewardedVideoAdapterRouter.h"
+#import "ISVungleAdapter.h"
+
+@interface ISVungleRewardedVideoAdapterRouter()
+
+@property (nonatomic, strong, nullable) NSString *bidPayload;
+
+@end
+
+@implementation ISVungleRewardedVideoAdapterRouter
+
+- (instancetype)initWithPlacementID:(NSString *)placementID
+                      parentAdapter:(ISVungleAdapter *)parentAdapter
+                           delegate:(id<ISRewardedVideoAdapterDelegate>)delegate {
+    if (self = [super init]) {
+        _placementID = placementID;
+        _parentAdapter = parentAdapter;
+        _delegate = delegate;
+        _bidPayload = nil;
+    }
+
+    return self;
+}
+
+- (void)loadRewardedVideoAd {
+    self.rewardedVideoAd = [[VungleRewarded alloc] initWithPlacementId:self.placementID];
+    self.rewardedVideoAd.delegate = self;
+
+    if ([self.rewardedVideoAd canPlayAd]) {
+        LogInternal_Internal(@"Rewarded ad: %@ is loaded", self.placementID);
+        [self.delegate adapterRewardedVideoHasChangedAvailability:YES];
+        return;
+    }
```

Comment:
L36 - L40 is not needed here because there is no caching right now. So it will never be ready to play after making the new ad instance.

Reply:
+1